odm2api/ODM2/services/readService.py

```diff
@@ -1347,6 +1349,8 @@ def getResultValues(self, resultids, starttime=None, endtime=None):
             resultids (list): List of SamplingFeatureIDs.
             starttime (object, optional): Start time to filter by as datetime object.
             endtime (object, optional): End time to filter by as datetime object.
+            lowercols (bool, optional): Make column names to be lowercase.
+                                        Default to True.
```

Comment:
Add this at the end of the `lowercols` description: "Please start upgrading your code to rely on CamelCase column names. In a near-future release, the default will be changed to False, and later the parameter may be removed.